net-wireless/unifi/unifi-5.6.37.ebuild

```diff
@@ -0,0 +1,77 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit systemd user
+
+DESCRIPTION="A Management Controller for Ubiquiti Networks UniFi APs"
+HOMEPAGE="https://www.ubnt.com"
+SRC_URI="https://dl.ubnt.com/unifi/${PV}/UniFi.unix.zip -> ${P}.zip"
+
+KEYWORDS="-* ~amd64 ~x86"
+LICENSE="ubiquiti"
+SLOT="5/6"
+
+RDEPEND="dev-db/mongodb
+	dev-java/tomcat-native
+	virtual/jre:1.8"
+
+DEPEND="app-arch/unzip"
+
+RESTRICT="bindist mirror strip"
+
+S="${WORKDIR}/UniFi"
+
+DOCS=( "readme.txt" )
+
+QA_PREBUILT="usr/lib/unifi/lib/native/Linux/x86_64/*.so
+	usr/lib64/unifi/lib/native/Linux/x86_64/*.so"
+
+pkg_setup() {
+	enewgroup unifi
+	enewuser unifi -1 -1 /var/lib/unifi unifi
+}
+
+src_install() {
+	default
+
+	insinto /usr/$(get_libdir)/unifi
+	doins -r dl webapps
+
+	# Install MongoDB wrapper script, to avoid problems with >= 3.6.0
+	# See https://community.ubnt.com/t5/UniFi-Routing-Switching/MongoDB-3-6/td-p/2195435
+	if has_version ">=dev-db/mongodb-3.6.0"; then
+		exeinto /usr/$(get_libdir)/unifi/bin
+		doexe "${FILESDIR}"/mongod
+	else
+		dodir /usr/$(get_libdir)/unifi/bin
+		dosym ../../bin/mongod mongod
+	fi
+
+	insinto /usr/$(get_libdir)/unifi/lib
+	doins lib/*.jar
+
+	exeinto /usr/$(get_libdir)/unifi/lib/native/Linux/x86_64
+	doexe lib/native/Linux/x86_64/*.so
+
+	for dir in conf run tmp work; do
+		keepdir /var/lib/unifi/${dir}
+	done
+	keepdir /var/lib/unifi
```

Comment:
Why? You've already kept all four subdirectories.

Reply:
Yep, you are right, removed.